injected/scripts/utils/build.js

```diff
@@ -36,92 +16,105 @@ const prefixMessage = '/*! © DuckDuckGo ContentScopeScripts protections https:/
  * @param {string} [params.name]
  * @return {Promise<string>}
  */
-export async function rollupScript(params) {
+export async function bundle(params) {
     const { scriptPath, platform, name, featureNames } = params;
 
-    const extensions = ['firefox', 'chrome', 'chrome-mv3'];
+    const extensions = ['firefox', 'chrome-mv3'];
     const isExtension = extensions.includes(platform);
     let trackerLookup = '$TRACKER_LOOKUP$';
     if (!isExtension) {
         const trackerLookupData = readFileSync('../build/tracker-lookup.json', 'utf8');
         trackerLookup = trackerLookupData;
     }
-    const suffixMessage = `/*# sourceURL=duckduckgo-privacy-protection.js?scope=${name} */`;
-    const plugins = [
-        css(),
-        svg({
-            stringify: true,
-        }),
-        loadFeatures(platform, featureNames),
-        resolve(),
-        commonjs(),
-        replace({
-            preventAssignment: true,
-            values: {
-                'import.meta.injectName': JSON.stringify(platform),
-                // To be replaced by the extension, but prevents tree shaking
-                'import.meta.trackerLookup': trackerLookup,
-            },
-        }),
-        prefixPlugin(prefixMessage),
-    ];
-
-    if (platform === 'firefox') {
-        plugins.push(suffixPlugin(suffixMessage));
-    }
+    const suffixMessage = platform === 'firefox' ? `/*# sourceURL=duckduckgo-privacy-protection.js?scope=${name} */` : '';
```

Comment:
I think we can remove this now, this was because we had several different injection sites so eased debugging but moving to mv3 that's no-longer the case.